packages/node_modules/@webex/internal-plugin-devices/src/constants.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable-next-line import/prefer-default-export */
+export const FEATURE_COLLECTION_NAMES = ['developer', 'entitlement', 'user'];
```

Comment:
Could each of those be exported as well?

```js
export const FEATURE_COLLECTION_DEVELOPER = 'developer';
export const FEATURE_COLLECTION_ENTITLEMENT = 'entitlement';
export const FEATURE_COLLECTION_USER = 'user';
export const FEATURE_COLLECTION_NAMES = [FEATURE_COLLECTION_DEVELOPER, FEATURE_COLLECTION_ENTITLEMENT, FEATURE_COLLECTION_USER];
```

Reply:
take care of!